astro-client/mocks/Client.go

```diff
@@ -323,3 +366,20 @@ func (_m *Client) UpdateDeployment(input *astro.DeploymentUpdateInput) (astro.De
 
 	return r0, r1
 }
+
+// TODO: why did this get generated?
```

Comment:
This is an auto-generated file created via mockery, you might be using the latest version of mockery, hence the mockConstructor & NewClient were added to the file.

Reply:
@neel-astro Is it ok to leave them in the generated output? Or should I downgrade to the correct version of mockery? 
What specific version?

Reply:
It's okay to leave it as it is since you were using the latest version, you can remove the TODO comments if you want to.

Reply:
will do thanks!